src/lib/index.jsx

```diff
@@ -8,7 +8,7 @@ export default function AddToCalendar(WrappedButton, WrappedDropdown) {
   return class AddToCalendarWrapped extends Component {
     static propTypes = {
       buttonProps: PropTypes.shape(),
-      buttonText: PropTypes.string,
+      buttonText: PropTypes.node,
```

Comment:
Since this no longer accepts _just_ text, maybe the prop name should change to reflect that. button _Text_ is no longer accurate.